contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,91 @@
+
+Welcome to Kubernetes! (New Developer Guide)
+============================================
+
+---
+
+_This document assumes that you know what Kubernetes does. If you don't, check
+out [the Kubernetes keynote](https://www.youtube.com/watch?v=8SvQqZNP6uo) and
+try the demo at [https://k8s.io/](https://k8s.io/)._
+
+Introduction
+------------
+
+Have you ever wanted to contribute to the coolest cloud technology? This
+document will help you understand the organization of the Kubernetes project and
+direct you to the best places to get started. By the end of this doc, you'll be
+able to pick up issues, write code to fix them, and get your work reviewed and
+merged.
+
+If you have questions about the development process, feel free to jump into our
+[Slack Channel](http://slack.k8s.io/) or join our [mailing
+list](https://groups.google.com/forum/#!forum/kubernetes-dev).
+
+Special Interest Groups
+-----------------------
+
+Kubernetes developers work in teams called Special Interest Groups (SIGs). At
+the time of this writing there are [31 SIGs](../../sig-list.md).
+
+The developers within each SIG have autonomy and ownership over that SIG's part
+of Kubernetes. SIGs organize themselves by submitting markdown design documents
+to the [kubernetes/community](https://github.com/kubernetes/community)
+repository. You can see an example of this planning process at
+[kubernetes/community#645](https://github.com/kubernetes/community/pull/645).
+Like everything else in Kubernetes, a SIG is an open, community, effort. Anybody
+is welcome to jump into a SIG and begin fixing issues, critiquing design
+proposals and reviewing code.
+
+Most people who visit the Kubernetes repository for the first time are
+bewildered by the thousands of [open
+issues](https://github.com/kubernetes/kubernetes/issues) in our main repository.
+But now that you know about SIGs, it's easy to filter by labels to see what's
+going on in a particular SIG. For more information about our issue system, check
+out
+[issues.md](https://github.com/kubernetes/community/blob/master/contributors/devel/issues.md).
+
+Downloading, Building, and Testing Kubernetes
+---------------------------------------------
+
+This guide is non-technical, so it does not cover the technical details of
+working Kubernetes. We have plenty of documentation available under
+[github.com/kubernetes/community/contributors/devel](https://github.com/kubernetes/community/tree/master/contributors/devel).
+Check out
+[development.md](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md)
+for more details.
+
+Pull-Request Process
+--------------------
+
+The pull-request process is documented in [pull-requests.md](pull-requests.md).
```

Comment:
```
Contributing to Kubernetes is done through pull requests to the various repos. This process is documented in detail [here](pull-requests.md).
```
Reading over this too, the one thing that might be worth adding back here is something about the CLA and linking to `(../../CLA.md)`. It's an important step for brand new contributors, and you really only need to deal with it once.